app.py

```
@@ -1,91 +1,130 @@
+    The results are cached for performance.
     """
+    try:
+        # Download the config file
+        config_path = hf_hub_download(repo_id=REPO_ID, filename=CONFIG_FILENAME)
+        with open(config_path, "r") as f:
+            config = yaml.safe_load(f)
+
+        # Download the model file
+        model_path = hf_hub_download(repo_id=REPO_ID, filename=MODEL_FILENAME)
+        model = joblib.load(model_path)
+        return config, model
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)